### DISCUSSION TITLE
Frequent hangs

### DISCUSSION THREAD

**Discussion by rhopping73** [netbrain/zwift#180] [Q&A]
I am on Fedora 41 with an NVIDIA GeForce 1660 Ti GPU. I am finding this container to be so promising. It pretty much worked for me out of the box. However, I am finding that it frequently hangs on me, to the point that I would not be comfortable starting a real ride with it yet. For example, if I simple Watch a rider, with 15-20 minutes it freezes.

Is there anything I can provide to help debug the hang?

**Comment by quietvoid** [comment]
If you still have video screenshots enabled, try disabling them.

**Reply by rhopping73** [reply]
Thanks for the reply. I do have video screenshots disabled. The problem occurs at random times. I have been in the route menu, shop, watching a rider, or in my own ride. I noticed today that it is the graphics that lock up. The sound continues. 

I also find that is typically not possible to close the application from the app icon in my toolbar or the corner of the Zwift window. I pop-up appears telling me that the app is unresponsive. Eventually, I jump to the shell and kill the process there.

I am running in Podman.

**Reply by netbrain** [reply]
Sounds like a hardware or driver issue. Try reducing the graphics quality
and resolution to its lowest setting and see if that works.

**Reply by rhopping73** [reply]
Thanks. That reminded me I had made some changes over the weekend to troubleshoot issues while using the nvidia driver. I have since switched back to nouveau but forgot to back out those changes.

I have now also enabled the wayland experimental option 🤞. I AM running at 4K, so if this does not work I will start to back that down.

**Reply by rhopping73** [reply]
The problems continued with the nouveau driver. I gave the nvidia driver another try. Running great now, with the wayland experimental option off. 

Thanks for the great project and the feedback about my question.

**Reply by rommeswi** [reply]
Is it like a 5 second freeze? I am also encountering that. But I just have integrated graphics so I always thought it was related to that.

**Reply by rhopping73** [reply]
I do still have the brief freeze from time to time, but it does not completely hang like it did when I was using the nouveau driver. That being said, I paused my Zwift account a couple months ago, it’s too nice to be inside now. 😁 I’ll be back in December.
